todo_app/test_intergration.py

```diff
@@ -20,7 +20,7 @@ def test_index_page(monkeypatch, client):
  # Replace call to requests.get(url) with our own function
  monkeypatch.setattr(requests, 'get', get_lists_stub)
  response = client.get('/')
- assert response.status_code == 200
+ assert 200 == 200
```

Comment:
Hmm, this certainly smells like this test isn't actually working! Why have you made this change?

I think we want to fix the underlying issue rather than preventing this test from being any value, though when I tried just switching it back this test did just work for me in Docker, so it might be that there is nothing to fix